todo_app/templates/index.html

```diff
@@ -1,3 +1,8 @@
+{% if view_model.user_role == 'writer' %}
```

Comment:
This might tell writers they can write, but it's not much good for telling non-writers that they can't which is quite confusing!

Instead, what we probably want to do is only show the appropriate options to each person, e.g. for your "new item" form then we only want to show it to writers:
```
 {% if view_model.user_role == 'writer' %}
  <form action= "/createnewcard" method="post" > 
    <input type="text" name = "title">
    <input type="submit" value= "Add your new item in the box">
  </form>
  {% endif %}
```

Reply:
done, also added to /complete_item